### ISSUE TITLE
Generate an alert if the API query fails

### ISSUE THREAD

**Issue by jaxxstorm** [Yelp/sensu_handlers#31]
Hi guys,

This is more of a discussion than anything, just wondering if maybe you have any thoughts.

I have a monkey patched version of your JIRA handler, but in our orgs infinite wisdom, changes to transitions, required custom fields or such like will often mean the handler fails. It's difficult to notice this unless you're active catching the exceptions and checking events vs open jira tickets.

Can you think of a way whereby a sensu alert (with another handler) is generated without having to pull in a bunch of existing gems?

Obviously we can rescue the exception and go from there, just wondering if you have any novel ideas..

**Comment by solarkennedy** [comment]
Hmmmm. That is an interesting idea.
We also face similar issues at Yelp with custom Projects with custom needs.

To me the easiest way to handle this is not to invoke more handles, but instead send another event to the localhost:3030 socket? (with like, notification_email => 'somebody_who_knows_how_to_fix')

**Comment by jaxxstorm** [comment]
Yes I think that's probably it! Shouldn't need any additional gems and we just open a TCP socket. I'll see if I can knock something together.
